### PR TITLE
Setattr

### PR DESCRIPTION
**Pull request by yyoudlm**
Add chown method in HCFSFuseFileSystem.java

### CODE REVIEW

File: src/main/java/hcfsfuse/fuse/HCFSFuseFileSystem.java [hunk 1]
```diff
@@ -97,6 +104,52 @@ public HCFSFuseFileSystem(FileSystem fs, FuseOptions fuseOptions,
...
+        groupName = AlluxioFuseUtils.getGroupName(userName);
+      }
+
+      if (userName.isEmpty() && groupName.isEmpty()) {
+        // This should never be reached
+        LOG.info("Unable to change owner and group of file {} when uid is {} and gid is {}", path,
+                userName, groupName);
+      } else if (userName.isEmpty()) {
+        LOG.info("Change group of file {} to {}", path, groupName);
+        mFileSystem.setOwner(turi, userName, groupName);
```
Comment (maobaolong) [inline src/main/java/hcfsfuse/fuse/HCFSFuseFileSystem.java]: if `username` is empty , what will happen?

Comment (yyoudlm) [reply src/main/java/hcfsfuse/fuse/HCFSFuseFileSystem.java]: This is normal. One of the users or groups is allowed to be empty in chown

Comment (maobaolong) [reply src/main/java/hcfsfuse/fuse/HCFSFuseFileSystem.java]: I mean that if users or groups is `null`, the original username or group name remains unchanged. But `""` is empty, so it mean change username or groupname to `""` empty, any thoughts?

**Comment by maobaolong** [comment]
@yyoudlm Thank you for your improvement. LGTM overall, Merge after CI checks getting green.

**Pull request merged**